deltalayer/provider.tf

```diff
@@ -0,0 +1,11 @@
+provider "google" {
+  alias = "target"
+}
+
+provider "google" {
+  alias = "dns"
+}
+
+provider "google-beta" {
+  alias = "target"
+}
```

Comment:
I would delete this entirely. This tf module isn't creating anything that requires google-beta